Libs/Gst/Classes/Parse.cs

```diff
@@ -10,16 +10,24 @@ public static class Parse
         /// <param name="pipelineDescription">Description of the pipeline</param>
         /// <returns>A new element</returns>
         /// <exception cref="GLib.GException">Throws an exception in case of an error</exception>
-        public static Element Launch(string pipelineDescription)
+        public static Element? Launch(string pipelineDescription)
         {
-            //TODO: As this method wrapps a global function it should probably be part of the toolkit layer.
-            
             IntPtr ret = Global.Native.parse_launch(pipelineDescription, out IntPtr error);
 
             if (error != IntPtr.Zero)
                 throw new GLib.GException(error);
 
-            return new Element(ret); //Parse always returns a new object
+            return GObject.Object.WrapHandle<Element>(ret);
+        }
+
+        public static Element? BinFromDescription(string binDescription, bool ghostUnlinkedPads)
+        {
+            IntPtr ret = Global.Native.parse_bin_from_description(binDescription, ghostUnlinkedPads, out IntPtr error);
+            
+            if (error != IntPtr.Zero)
+                GLib.Error.HandleError(error);
+            
+            return GObject.Object.TryWrapHandle<Element>(ret, out Element? element) ? element : null;
```

Comment:
This should probably not return null but throw an exception to be in line with `Launch`.

I would suggest that we are throwing exceptions in the "standard" case for methods with an error. In case we don't want to throw I would suggest `TryXXX` as a name with a bool return and an `out` parameter to make the behaviour of the methods absolutely clear and to be able to differentiate them.

In this method there are sometimes exceptions thrown and sometimes a `null` is returned which should not be possible as it makes using the API very hard.

Reply:
Fixed, see above.